src/fontra/views/editor/panel-related-glyphs.js

```diff
@@ -148,6 +148,7 @@ export default class RelatedGlyphPanel extends Panel {
 
     for (const item of this.accordion.items) {
       this._updateAccordionItem(item, glyphName, codePoint).then((hasResult) => {
+        this.accordion.showHideAccordionItem(item, hasResult);
```

Comment:
Why is this done in a different place than the original `parent.hidden = hideAccordionItem`? If it was done in `_updateAccordionItem()` before, I think it should still b done there, or you'll change the bahavior.

Reply:
Ok, I now see that it is equivalent, because `_updateAccordionItem()` is only called in one spot. Perhaps it is better this way.